### PR TITLE
Update gh workflow with ghcr

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: .github/workflows/qa_build.yaml [hunk 1]
```diff
@@ -47,6 +47,22 @@ jobs:
...
+            ghcr.io/${{ github.repository }}
+            ${{ github.repository }}
+          tags: |
+            type=sha
+      - name: login to ghcr
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```
Comment (jerith) [inline .github/workflows/qa_build.yaml:78]: Maybe put this in a separate job? That will run it in parallel with the dockerhub build and also avoid any issues with leftover docker state and such.

Comment (erikh360) [reply .github/workflows/qa_build.yaml:78]: Good idea

**Comment by erikh360** [comment]
@jerith ready for another look

**Review by jerith — APPROVED** [review]
:+1:

**Pull request merged**